wagtail/admin/templates/wagtailadmin/pages/action_menu/save_draft.html

```diff
@@ -0,0 +1,2 @@
+{% load i18n %}
+<button type="submit" class="button action-save button-longrunning {% if is_revision %}warning{% endif %}" data-clicked-text="{% trans 'Saving…' %}" {% if page.locked %}disabled {% endif %}><span class="icon icon-spinner"></span><em>{% if page.locked %}{% trans 'Page locked' %}{% else %}{% if is_revision %}{% trans 'Replace current draft' %}{% else %}{% trans 'Save draft' %}{% endif %}{% endif %}</em></button>
```

Comment:
Looks like this was copied from `edit.html`. Does it work on the create view? (just wondering if the `{% if page.locked %}` bit is a problem as the other button templates don't seem to have this).

Reply:
The `{% if page.locked %}` checks shouldn't be a problem for the create view, as `page.locked` is always false for a newly-created page.

However, the current behaviour isn't quite right on the edit page when the page is locked, because `SaveDraftMenuItem` now hides itself for locked pages (the default behaviour of `ActionMenuItem.is_shown`), as do all other menu items, so the menu disappears completely. In any case, these template-level customisations to the "Save draft" button (e.g. changing its label to "Page locked") are only meaningful when it's the default button, and the whole point of this PR is that we can't assume that any more :-)

I think the most logical solution is to make the "Page Locked" button a menu item in its own right - only ever shown when the page _is_ locked, of course - and give it a very high (or low, depending on the outcome of https://github.com/wagtail/wagtail/pull/5500#pullrequestreview-286868977) order value so that if another custom button defines itself as active on locked pages for some reason, it won't become the default (unless the developer is really intent on doing so) and so the "Page Locked" label won't end up buried in the list.

Reply:
Above proposal now implemented in https://github.com/gasman/wagtail/commit/5abcddfa478d526632276c4c54af0716e72246f7